lib/test.py

```diff
@@ -94,7 +94,8 @@ def flush(self):
 
 
 def get_filename_by_test(postfix, test_name):
-    rg = re.compile(r'\.test.*')
+    """For <..>/<name>_test.* or <..>/<name>.test.* return <name> + postfix"""
```

Comment:
I would just give a couple of examples:

- .result, foo/bar.test.lua -> bar.result
- .reject, bar_test.lua -> bar.reject